CHANGELOG.md

```diff
@@ -18,7 +18,7 @@ FEATURES:
 * `resource/vcd_org_vdc` added Org VDC update and full state read - [GH-275]
 * `resource/vcd_org_vdc` added Org VDC metadata support - [GH-276]
 * `resource/vcd_snat` added ability to choose network name and type. [GH-282] 
-* `resource/vcd_dnat` added ability to choose network name and type. [GH-282]
+* `resource/vcd_dnat` added ability to choose network name and type. [GH-282] [GH-292] [GH-293]
```

Comment:
Never tested before, but should be like that as per https://www.terraform.io/docs/extend/best-practices/versioning.html
> IMPROVEMENTS:
> * resource/load_balancer: Add `ATTRIBUTE` argument (support X new functionality) [GH-12]
> * resource/subnet: Now better [GH-22, GH-32]

```suggestion
* `resource/vcd_dnat` added ability to choose network name and type. [GH-282, GH-292, GH-293]
```

Reply:
changed